src/package.json

```diff
@@ -140,7 +140,9 @@
     "vue-router": "4.5.0",
     "vue-tsc": "2.2.8",
     "workbox-precaching": "7.3.0",
-    "zod": "3.24.2"
+    "zod": "3.24.2",
+    "semantic-release": "24.2.3",
+    "@vueuse/core": "^12.7.0"
```

Comment:
Caret detected. Just run `pnpm add -DE @vueuse/core` in the `src` directory (after removing these new lines here) and you should be good to go.